gradle/distributionTesting.gradle

```diff
@@ -106,6 +107,15 @@ distributionTestTasks.all { DistributionTest task ->
     }
 }
 
+def collectMirrorUrls() {
+    // expected env var format: repo1_id:repo1_url,repo2_id:repo2_url,...
+    def mirrors = System.getenv().get('REPO_MIRROR_URLS')
+    !mirrors ? [:] : mirrors.split(',').collectEntries { nameToUrl ->
```

Comment:
⭕️  It took me a little longer to decode the conditional logic in `!mirrors ? [:] : mirrors.split...`, not a huge deal but I think null-safe navigation followed by null coalescing might be easier to parse, I would even ditch the local variable so the whole method becomes a single expression:

```
System.getenv('REPO_MIRROR_URLS')?.split(',')?.collectEntries { nameToUrl ->
    ...
} ?: [:]
```

Reply:
Yes, that's much more readable. I've fixed that.